p2p/vbitvec.go

```diff
@@ -127,17 +135,17 @@ func (v *VBitVec) has1fecBit(s *VBitVec) bool {
     return diffCnt == 1
 }
 
-//getBitsCnt calculate bit num
+// getBitsCnt Calculate bit num
```

Comment:
Commens should be a sentence,eg: getBitsCnt calculates bit num

Reply:
nice suggestion.